common/app/layout/Sublink.scala

```diff
@@ -303,5 +307,51 @@ case class ContentCard(
     case _ => false
   }
 }
+object ContentCard {
 
+  def makeFromApiContent(apiContent: contentapi.Content): Option[ContentCard] = {
+
+    apiContent.fields.map { fields =>
+      val discussionSettings = for {
+        commentable <- fields.commentable
+        isClosedForComments <- fields.commentCloseDate.map(closeDate => new DateTime(closeDate.dateTime).isBeforeNow)
+      } yield DiscussionSettings(commentable, isClosedForComments, None)
+
+      val isVideo = apiContent.`type` == ContentType.Video
+      val faciaCardHeader = FaciaCardHeader(false, false, isVideo, false, false, None, apiContent.webTitle, EditionalisedLink(apiContent.webUrl))
```

Comment:
do you mind using named parameters for all of these and some other vals below (and probably put them on newlines) - I think that tends to be our convention for boolean parameters at least